api/v1alpha2/virtualmachine_types.go

```diff
@@ -10,6 +10,28 @@ import (
 	"github.com/vmware-tanzu/vm-operator/api/v1alpha2/common"
 )
 
+const (
+	// VirtualMachineMetadataReadyCondition documents the state of the metadata passed to the VirtualMachine.
+	VirtualMachineMetadataReadyCondition = "VirtualMachineMetadataReady"
```

Comment:
I don't think we have a concept of "metadata" in v1a2?  It is called `VirtualMachineBootstrapSpec`.  Or maybe this is more nuanced and I am missing something?